test/OrchardCore.Tests/Shell/ShellHostTests.cs

```diff
@@ -0,0 +1,95 @@
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Environment.Shell.Models;
+using OrchardCore.Tests.Apis.Context;
+using Xunit;
+
+namespace OrchardCore.Tests.Shell;
+public class ShellHostTests : SiteContext
+{
+    public static IShellHost ShellHost { get; }
+
+    static ShellHostTests()
+    {
+        ShellHost = Site.Services.GetRequiredService<IShellHost>();
+    }
+
+    [Fact]
+    public static async Task FindCaseInsensitiveShellSettings()
```

Comment:
Can we use `Theory` for the first two unit tests while they are similar?

Reply:
@hishamco  Can you please explain why you are requesting to use theory? It's my understanding that `[Thoery]` would be used when there are parameters involved and `[Fact]` where there are no parameters.

Reply:
I think I understood your request. Please check the updated tests